eureka-client/src/main/java/com/netflix/appinfo/InstanceInfo.java

```diff
@@ -1153,4 +1153,30 @@ private static String resolveDeploymentContextBasedVipAddresses(
         return result;
     }
 
+    /**
+     * Get the zone that a particular instance is in.
+     * Note that for AWS deployments, myInfo should contain AWS dataCenterInfo which should contain
+     * the AWS zone of the instance, and availZones is ignored.
+     *
+     * @param availZones the list of available zones for non-AWS deployments
+     * @param myInfo
+     *            - The InstanceInfo object of the instance.
+     * @return - The zone in which the particular instance belongs to.
+     */
+    public static String getZone(String[] availZones, InstanceInfo myInfo) {
```

Comment:
That's true, it could be. It was mostly because this was just a transport of the method from the client.